src/components/CommentVotingButton.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+
+const CommentVotingButton = ({ comment, setCommentVotes, commentVotes }) => {
+  //   const [thisComment, setThisComment] = useState(comment);
```

Comment:
Could you remove your commented out code. Its the worst thing to leave in as no one else will remove it, because they think someone else probably needs it